packit_service/worker/tasks.py

```diff
@@ -75,6 +84,8 @@
 from packit_service.worker.jobs import SteveJobs
 from packit_service.worker.result import TaskResults
 
+_CACHE_MAXSIZE = 100
```

Comment:
I don't see this to be used anywhere

Reply:
It was an old value, I don't know why `black` didn't complain. I will try to remove it.